README.md

```diff
@@ -1,5 +1,288 @@
-# FCC Test Suite Template
+<a name="readme-top"></a>
 
-**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
 
-You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
+Modify this file to match your project and remove sections that don't apply.
+
```

Comment:
Kindly remove this section from your code, I don't think it should make up your README file considering the fact we want to make your project look professional 😉👍